src/common/base_test.py

```diff
@@ -56,7 +62,8 @@ def options(self) -> webdriver.ChromeOptions:
             'FencedFrames:implementation_type/mparch',
             'BiddingAndScoringDebugReportingAPI',
             'PrivacySandboxAdsAPIsOverride',
-            'OverridePrivacySandboxSettingsLocalTesting'
+            'OverridePrivacySandboxSettingsLocalTesting',
+            'PrivacySandboxAttestationsHigherComponentRegistrationPriority',
```

Comment:
This isn't required for `--privacy-sandbox-enrollment-overrides` (AFAIU this flag works event without attestations component loaded), but it is required for `updatePrivacySandboxAttestationsComponent` so I'd leave it here.